aws/internal/keyvaluetags/generators/updatetags/main.go

```diff
@@ -130,6 +131,7 @@ func main() {
 		"UntagFunction":                   ServiceUntagFunction,
 		"UntagInputRequiresTagType":       ServiceUntagInputRequiresTagType,
 		"UntagInputTagsField":             ServiceUntagInputTagsField,
+		"UntagInputKeysType":              ServiceUntagInputRequiresKeyType,
```

Comment:
Nit: to match the other generator functions for consistency

```suggestion
		"UntagInputRequiresKeyType":       ServiceUntagInputRequiresKeyType,
```